docs/user_manual/install.md

```diff
@@ -0,0 +1,75 @@
+# Installation Instructions
+
+## Docker Installation
+
+Docker can be used to run the software without having to complete the heasoft/caldb/wcstools installation procedure yourself (or to run on windows). The image can be found on [docker hub](https://hub.docker.com/repository/docker/dresscodeswift/dresscode). It also includes the latest version of the DRESSCode pipeline, along with any other python dependencies.
+
+To download and open an interactive shell with the prerequisits already installed:
+
+```sh
+docker pull dresscodeswift/dresscode
+docker run --rm -it dresscodeswift/dresscode /bin/bash
+```
+
+From there run the pipeline, which is located in `/opt/dresscode`
+
+You will probably want to download the data onto a persistent volume. For this, see [docker documentation](https://docs.docker.com/storage/volumes/).
+
+## Local Installation
+
+The following steps will install Heasoft, caldb, and wcstools on your machine
+
+### Download the software
```

Comment:
Probably best to say Heasoft software (to distinguish with following steps)

Reply:
yea, this PR was to essentially copy over the text from the user manual. we can do some minor updates, but I'd prefer to do any re-writing of the docs in a separate PR.